cloudformation/media-atoms-dynamo-code.json

```diff
@@ -15,8 +15,12 @@
             "Description": "Code audit dynamo table",
             "Type": "String",
             "Default": "audit-media-atom-maker-CODE"
+        },
+        "UploadTrackingAtomMakerCodeTable": {
```

Comment:
@jennysivapalan if we intend to move them in the future, would it make sense to do that now while the tool is lightly used? (so we could possibly copy data without downtime)